tests/foreman/smoke/test_api_smoke.py

```diff
@@ -879,22 +879,24 @@ def test_smoke(self):
         ).create()
 
         # step 2.8: Associate YUM repository to new content view
-        content_view.set_repository_ids([repo1.id])
+        content_view.repository = [repo1]
```

Comment:
so in general, does this append the repo to content view or replace existing repos with the supplied one?